public/Javascript/script.js

```diff
@@ -0,0 +1,310 @@
+/*background colors do not reset!
+tie game doesn't register
+*/
+//Elements that need to be targeted
+let start = document.getElementById("start");
+let cells = Array.from(document.getElementsByClassName("cell"));
+let playerOneSelect = document.getElementById("playerOne");
+let playerTwoSelect = document.getElementById("playerTwo");
+let clock = document.getElementById("clock");
+let computerButton = document.getElementById('computerPlayer');
+//Global variable
+
+let playerOne = "X";
+let playerTwo = "O";
+let playerTurn = playerOne;
+let interval;
+let clockCount = 0;
+let moveCounter = 0;
+let AI = false;
+
+
+
+//Every move adds to the moves and checks for a win. If it is more than 8 moves, declares a draw.
+function moves() {
+  if (moveCounter < 8) {
+    checkWin();
+  } else if ((moveCounter = 8)) {
+    statusArea.textContent = "It's a Draw";
+    boardReset();
+  }
+  moveCounter++;
+}
```

Comment:
There's a common error written into here that's causing your draw function to not work correctly and changing just a few things will fix it